lib/ice_cube.rb

```diff
@@ -68,7 +71,7 @@ module Validations
   # Defines the format used by IceCube when printing out Schedule#to_s.
   # Defaults to '%B %e, %Y'
   def self.to_s_time_format
-    @to_s_time_format ||= '%B %e, %Y'
+    I18n.t("ice_cube.date.formats.default")
```

Comment:
You could always change `I18n.t("ice_cube.date.formats.default")` to override that :smile:, that is actually my proposed way to do that. What do you think? But maybe there are some docs / other methods to be changed accordingly?